rpm.mk

```diff
@@ -79,18 +79,9 @@ ifeq ($(DOKKU_VERSION),master)
 else
 	echo $(DOKKU_VERSION) > /tmp/build/var/lib/dokku/VERSION
 endif
-	cat /tmp/build/var/lib/dokku/VERSION | cut -d '-' -f 1 | cut -d 'v' -f 2 > /tmp/build/var/lib/dokku/STABLE_VERSION
-ifneq (,$(findstring false,$(IS_RELEASE)))
-	sed -i.bak -e "s/^/`date +%s`-/" /tmp/build/var/lib/dokku/STABLE_VERSION && rm /tmp/build/var/lib/dokku/STABLE_VERSION.bak
-endif
-ifdef DOKKU_GIT_REV
-	echo "$(DOKKU_GIT_REV)" > /tmp/build/var/lib/dokku/GIT_REV
-else
-	git rev-parse HEAD > /tmp/build/var/lib/dokku/GIT_REV
-endif
 
 	@echo "-> Creating rpm package"
-	VERSION=$$(cat /tmp/build/var/lib/dokku/STABLE_VERSION); \
+	VERSION=$$(cat /tmp/build/var/lib/dokku/VERSION); \
```

Comment:
IIRC a valid deb version is not a valid rpm version. That was the driver around having a `STABLE_VERSION` and a `VERSION` file (though I could be wrong).

Reply:
Should we instead do `git describe --tags` for the version, as those are valid for "upstream" and rpm, and transform them into valid deb version? It is trivial to go from a valid upstream/rpm version to a valid debian one, but a tad harder the other way around.

Reply:
Sounds like a good strategy.